Web Sevice 1 (Soap)/TempratureConverterClient/src/com/metacube/TempratureConverterServiceLocator.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * TempratureConverterServiceLocator.java
+ *
+ * This file was auto-generated from WSDL
+ * by the Apache Axis 1.4 Apr 22, 2006 (06:55:48 PDT) WSDL2Java emitter.
+ */
+
+package com.metacube;
+
+public class TempratureConverterServiceLocator extends org.apache.axis.client.Service implements com.metacube.TempratureConverterService {
+
+    public TempratureConverterServiceLocator() {
+    }
+
+
+    public TempratureConverterServiceLocator(org.apache.axis.EngineConfiguration config) {
+        super(config);
+    }
+
+    public TempratureConverterServiceLocator(java.lang.String wsdlLoc, javax.xml.namespace.QName sName) throws javax.xml.rpc.ServiceException {
+        super(wsdlLoc, sName);
+    }
+
+    // Use to get a proxy class for TempratureConverter
+    private java.lang.String TempratureConverter_address = "http://localhost:8080/TempratureConverter/services/TempratureConverter";
+
+    public java.lang.String getTempratureConverterAddress() {
+        return TempratureConverter_address;
+    }
+
+    // The WSDD service name defaults to the port name.
+    private java.lang.String TempratureConverterWSDDServiceName = "TempratureConverter";
+
+    public java.lang.String getTempratureConverterWSDDServiceName() {
```

Comment:
you do not need to use fully qualified name like java.lang.String. Read this link
https://coderanch.com/t/390734/java/String-java-lang-String-declaration